tests/test_pca.py

```diff
@@ -103,6 +103,7 @@ def test_pca_randomized_solver():
     )
 
 
+@pytest.mark.skip(reason="Passing None has been deprecated")
```

Comment:
I would rather if we just remove the pytest.warns(None) call and assert this differently instead of skipping the complete test

Reply:
Done https://github.com/dask/dask-ml/pull/980/commits/73014db38788f5ba4f04a6d3ed157c19dd0b0961